concepts.md

```diff
@@ -207,6 +207,16 @@ Each half contains the name of the authentication method, a value (in case of em
 
 A [Public Provisional Identity] can be generated from a [Secret Provisional Identity] and consists of the [Secret Provisional Identity] without its secrets parts.
 
+### TLS
+
+Transport Layer Security is always used between *Tanker Core* and *Tanker Server* to exchange information.
+This guarantee secrecy of the data exchanged.
```

Comment:
`guarantees the confidentiality and authenticity`

Reply:
go fix wikipedia!